src/pages/Assistance/Assistance.tsx

```diff
@@ -160,6 +160,14 @@ const Assistance = () => {
     },
   });
 
+  useEffect(() => {
+    if (formik.dirty) {
+      registerUnloadEvent();
```

Comment:
you have to remove the automatica registraction of the unloadEvent in line 92 (useUnloadEventInterceptorAndActivate) and here set the title and description configured there